addons/udes_stock/data/product_categories.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<odoo>
+  <data>
+
+    <!-- Construct "Height" product category -->
+    <record id="product_category_height" model="product.category">
```

Comment:
Q: Why do we have height categories but not length or width?

Reply:
I think it is because in UDES 11 some customers (hds, anniversary) want height categories, but not length/width categories. The height categories are only used by specific customers.

Reply:
It is only used for specific customers. Some customer modules use the height category, I assume those customers do not want length or width categories.

Reply:
Just for the benefit of anyone who checks after this is removed, the "Height" category is actually meant to reference the pallet height.

But isn't needed in v14 for now, so will be removed as Jack has said elsewhere. :)